domain/src/DomainForm.php

```diff
@@ -117,7 +117,13 @@ public function save(array $form, FormStateInterface $form_state) {
     else {
       drupal_set_message($this->t('Domain record updated.'));
     }
+
+    if ($domain->isDefault()) {
```

Comment:
I suspect we want to clear the render cache when any domain changes, just to be safe.

We may need to do that for aliases records as well.